### DISCUSSION TITLE
Component wire:model

### DISCUSSION THREAD

**Discussion by abdosaeedelhassan** [livewire/livewire#5116] [3. Feature ideas]
Hi, sometimes we develop child component inside parent component, the child component may be custom input that needed as a property in parent component, parent component may be crud form, i suggest that if there a way to bind child component value to parent component property directly with wire:model directive instead of using events.

Thank you.

**Comment by joshhanley** [comment]
@abdosaeedelhassan thanks for the suggestion! Ideally that would be great if it could work that way. We're still trying to find ways to make that work without causing race conditions. The main issue is both the Livewire parent and child component make requests to the server, so they would have to be kept in sync some how.

But I generally would recommend not using a Livewire component for a custom input. Instead I would make a Blade component for the custom input that you can use in your main Livewire component that contains the crud form. That way you don't need to worry about keeping seperate components in sync as you only have one component that controls all the data on the page.

Hope this helps!

**Reply by abdosaeedelhassan** [reply]
Thank you for your recommendation

**Reply by joshhanley** [reply]
@abdosaeedelhassan no worries! 🙂
